lib/file.js

```diff
@@ -1,35 +1,30 @@
-// File
-// ====
-//
-// File object used for tracking files in `file-list.js`
+'use strict'
 
-// Dependencies
-// ------------
+/**
```

Comment:
I think it is good idea to convert comments to JSDoc, so they are picked up by IDE and show info on methods/classes. Do you think it is something worths doing?

Reply:
Ok by me, but in a separate PR if you want to do it.